DTS-STN/Alpha-Site
# Added/updated metadata

# Description

[Add the missing metadata to the pages](https://jira-dev.bdm-dev.dts-stn.com/browse/SCL-578)

The purpose of this PR is to add/update metadata on the pages that did not originally have metadata listed in the master content inventory in SharePoint when #406 was completed (I had temporary metadata on these pages but did not have official wording until now). These pages include `/500`, `/error`, `/404`, `/notsupported`, `/projects/virtual-assistant`, `/projects/digital-centre`, `/signup-info`, and the splash page.

In addition the above, I also updated every other page's metadata as per the master content inventory in SharePoint.

## Test Instructions

1. Pull in branch
2. Type npm run dev
3. Go to any page and open the dev console
4. Expand <head> to see the metadata


## Checklist


## Product and Sprint Backlog

[Jira](https://jira-dev.bdm-dev.dts-stn.com/secure/RapidBoard.jspa?rapidView=96&projectKey=SCL)
